src/replication_handler/replication_handler.cpp

```diff
@@ -198,6 +198,7 @@ bool ReplicationHandler::SetReplicationRoleReplica(const memgraph::replication::
                                                    const std::optional<utils::UUID> &main_uuid) {
   // We don't want to restart the server if we're already a REPLICA
```

Comment:
if you log, comment probably not needed

Reply:
This will be handled in this PR: https://github.com/memgraph/memgraph/pull/1711